tests/utils/autoTester.js

```diff
@@ -102,7 +102,7 @@ var testOver = function() {
     currentTestName = "";
     currentTestCase = null;
     
-    if (testMode == "manual") {
+    if (testMode === "manual" || testMode === "auto") {
```

Comment:
Aren't these the only two options?

Reply:
No - I am now working on a recursive option.  This will run all tests in all sub-folders.